src/popup/components/Modals/AccountCreate.vue

```diff
@@ -40,19 +50,81 @@ export default {
   props: {
     resolve: { type: Function, required: true },
   },
-  data: () => ({
-    PlusCircleIcon,
-    loading: false,
-  }),
-  methods: {
-    async createAccount() {
-      this.loading = true;
-      await this.$store.dispatch('accounts/hdWallet/create');
-      this.loading = false;
-      this.resolve();
-    },
+  setup(props, { root }) {
+    const plusCircleIcon = ref(PlusCircleIcon);
+    const loading = ref(false);
+    const multisigProgress = ref<MultisigCreationStep>('compiled');
+
+    async function createPlainOldAccount() {
+      loading.value = true;
+      await root.$store.dispatch('accounts/hdWallet/create');
+      loading.value = false;
+      (props.resolve as Function)();
+    }
+    async function createMultisigAccount() {
+      loading.value = true;
+      // eslint-disable-next-line no-use-before-define
+      await deployMultisigAccount(2, ['ak_aWUod4pwwhGmBLF3AYEpfm3SYQtsEJBuncTkGWmW7sf2f84My', 'ak_2cWZgLBL4rRgpiCoWVwwtDCJJFiTeETprvZsshEEYDQjdKyLpK']);
```

Comment:
Why are those accounts hardcoded?

Reply:
It's just a placeholder for the UI team to integrate. fetch it from the forms and insert it!